Gruntfile.js

```diff
@@ -17,12 +22,16 @@ module.exports = function(grunt) {
         src: components,
         dest: 'js/components.js',
       },
+      libcomponents: {
+        src: libcomponents,
+        dest: 'libtextsecure/components.js',
+      },
       curve25519: {
         src: [
           'build/curve25519_compiled.js',
           'build/curve25519.js',
         ],
-        dest: 'js/curve25519_compiled.js',
+        dest: 'libtextsecure/curve25519_concat.js',
```

Comment:
Seems like maybe you could skip this intermediate concat and add these files to concat:libtextsecure directly?